stacks/environment-aws/eks.tf

```diff
@@ -72,6 +90,15 @@ EOF
 data "aws_availability_zones" "available" {
 }
 
+data "template_file" "ondemand_toleration" {
```

Comment:
does the toleration FORCE the schedule to use the ondemand instance?  seems like may node affinity might be needed?  or, anti-affinity on spot?

Reply:
Nice catch.  The toleration will just allow the pod to be scheduled on the tainted ondemand instances, but it could still be potentially scheduled onto a spot instance.  Added an affinity to this template to force the scheduling to an ondemand instance.